usr/lib/common/obj_mgr.c

```diff
@@ -949,9 +949,9 @@ CK_RV object_mgr_find_in_map1(STDLL_TokData_t *tokdata,
         }
 
         rc = object_unlock(obj);
+        locked = FALSE;
         if (rc != CKR_OK)
```

Comment:
Commit msg above:
When object_unlock() fails, do not attempt  **to** unlock the object again

Reply:
will fix.